docs/make.bat

```diff
@@ -4,6 +4,9 @@ pushd %~dp0
 
 REM Command file for Sphinx documentation
 
+sphinx-apidoc -f -M -T -o modules ../cea
+print(Sphinx has updated rst files)
```

Comment:
did you check this? i'm not sure it can work.

Reply:
yep. batch doesn't have a `print(...)` statement. replace with:
```
echo Sphinx has updated rst files
```

Reply:
no worries ill have a look at these now